RotaryWheelControl/RotaryWheel.xaml.cs

```diff
@@ -143,6 +145,52 @@ public RotaryWheel()
             };
         }
 
+        private void SpinTo(int itemIndex, int durationInSec = -1)
+        {
+            Random r = new Random(DateTime.Now.Millisecond);
+            var angleFromYAxis = 360 - Angle;
+            SelectedItem = _pieSlices
+                .SingleOrDefault(p => p.StartAngle <= angleFromYAxis && (p.StartAngle + p.Angle) > angleFromYAxis);
+
+            int count = _pieSlices.Count;
+            int currIndex = _pieSlices.IndexOf(SelectedItem);
+            int fullSpin = itemIndex / count;
+            itemIndex = itemIndex % count;
+            int steps = currIndex-itemIndex;
+            if (steps < 0)
+                steps = count + steps;
+
+            var startAngle = SelectedItem.StartAngle + SelectedItem.Angle / 2;
+            var finalAngle = startAngle + fullSpin*360 + steps*360/count;
+
+            doubleAnimation.From = startAngle;
+            doubleAnimation.To = finalAngle;
+            if(durationInSec>0)
+                doubleAnimation.Duration = new Windows.UI.Xaml.Duration(new TimeSpan(0, 0, durationInSec));
```

Comment:
Can you use `TimeSpan.FromSeconds()` (https://msdn.microsoft.com/en-us/library/system.timespan.fromseconds(v=vs.110).aspx) instead?